utils/firacode-download.ps1

```diff
@@ -0,0 +1,83 @@
+# execute in the powershell console like this:
+# iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/srl-labs/containerlab/refs/heads/main/utils/firacode-download.ps1'))
+
+# Get the ID and security principal of the current user account
+$myWindowsID = [System.Security.Principal.WindowsIdentity]::GetCurrent()
+$myWindowsPrincipal = new-object System.Security.Principal.WindowsPrincipal($myWindowsID)
+# Get the security principal for the Administrator role
+$adminRole = [System.Security.Principal.WindowsBuiltInRole]::Administrator
+# Check to see if we are currently running "as Administrator"
+if ($myWindowsPrincipal.IsInRole($adminRole)) {
+    Write-Host "In"
```

Comment:
Change the message to say "In elevated prompt"... for some reason I just had it partially typed

Reply:
the full comment should be

"In an elevated prompt check to see if we are currently running "as Administrator""?